### ISSUE TITLE
dbExecute("USE test") raises error

### ISSUE THREAD

**Issue by krlmlr** [rstats-db/RMySQL#165]
about "prepared statement protocol", but doesn't in stable version.

**Comment by eddy85br** [comment]
\> `dbExecute(con, "use tests;")`
Error in eval(substitute(expr), envir, enclos) : 
  This command is not supported in the prepared statement protocol yet [1295]

\> `rs <- dbSendStatement(con, "use tests;")`
Error in eval(substitute(expr), envir, enclos) : 
  This command is not supported in the prepared statement protocol yet [1295]


`dpkg -l | grep 'lib.*client-dev'`
ii  libmariadbclient-dev                 5.5.49-1ubuntu0.14.04.1          amd64        MariaDB database development files

Is this a problem from my RMySQL or DBI packages or from MariaDB dev library?


```
sessionInfo()
R version 3.3.1 (2016-06-21)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 14.04.4 LTS

locale:
 [1] LC_CTYPE=en_US             LC_NUMERIC=C              
 [3] LC_TIME=pt_BR.UTF-8        LC_COLLATE=en_US          
 [5] LC_MONETARY=pt_BR.UTF-8    LC_MESSAGES=en_US         
 [7] LC_PAPER=pt_BR.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C            
[11] LC_MEASUREMENT=pt_BR.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] DBI_0.5-10         RMySQL_0.11-3      matrixStats_0.50.2 funModeling_1.2   
[5] Hmisc_3.17-4       ggplot2_2.1.0      Formula_1.2-1      survival_2.39-4   
[9] lattice_0.20-33   

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.7         RColorBrewer_1.1-2  git2r_0.15.0       
 [4] plyr_1.8.4          bitops_1.0-6        tools_3.3.1        
 [7] rpart_4.1-10        digest_0.6.10       memoise_1.0.0      
[10] tibble_1.2          gtable_0.2.0        Matrix_1.2-6       
[13] curl_2.0            gridExtra_2.2.1     stringr_1.1.0      
[16] withr_1.0.2         httr_1.2.1          dplyr_0.5.0        
[19] cluster_2.0.4       caTools_1.17.1      gtools_3.5.0       
[22] devtools_1.12.0     grid_3.3.1          nnet_7.3-12        
[25] data.table_1.9.6    R6_2.1.3            foreign_0.8-66     
[28] gdata_2.17.0        latticeExtra_0.6-28 reshape2_1.4.1     
[31] pander_0.6.0        ROCR_1.0-7          magrittr_1.5       
[34] scales_0.4.0        gplots_3.0.1        splines_3.3.1      
[37] assertthat_0.1      colorspace_1.2-6    KernSmooth_2.23-15 
[40] stringi_1.1.1       acepack_1.3-3.3     lazyeval_0.2.0     
[43] munsell_0.4.3       chron_2.3-47     

```

**Comment by krlmlr** [comment]
This is a RMySQL problem. Please use the "stable" version for now if you need this functionality.

Please also use dbSendStatement() and dbExecute() (as of DBI 0.5) to send statements that don't return data.

**Comment by eddy85br** [comment]
Thanks @krlmlr .
I was using the devel. version as an attempt to skip this issue: #40 

The problem was solved with the #40 "blind hack" commented by others.

**Comment by krlmlr** [comment]
Moved to rstats-db/RMariaDB (see link above).
